sequins.go

```diff
@@ -303,6 +303,11 @@ func (s *sequins) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if r.URL.Path == "/healthcheck" || r.URL.Path == "/healthz" {
```

Comment:
We should update the docs that you cannot have a db named healthcheck or healthz.